src/passes/replaceIdentifierContractMemberAccess.ts

```diff
@@ -0,0 +1,51 @@
+import { AST } from '../ast/ast';
+import { ASTMapper } from '../ast/mapper';
+
+import { ContractKind, Identifier, MemberAccess } from 'solc-typed-ast';
+import { CairoContract, CairoFunctionDefinition } from '../ast/cairoNodes';
+import { isExternallyVisible } from '../utils/utils';
+import { getContractTypeString } from '../utils/getTypeString';
+
+export class ReplaceIdentifierContractMemberAccess extends ASTMapper {
```

Comment:
Please add some documentation to this pass.

Reply:
Added it!